app/domain/output_document.rb

```diff
@@ -20,9 +20,11 @@ def attendee_name
     "#{appointment_summary.title} #{appointment_summary.first_name} #{appointment_summary.last_name}".squish
   end
 
-  def attendee_address
-    attendee_country = Countries.uk?(self.attendee_country) ? nil : self.attendee_country
+  def attendee_country
+    Countries.uk?(@appointment_summary.country) ? nil : @appointment_summary.country
```

Comment:
There is an [accessor method](https://github.com/guidance-guarantee-programme/output/blob/no-country-if-uk/app/domain/output_document.rb#L4) for `appointment_summary`. We should use that instead of the instance variable directly.

```rb
def attendee_country
  Countries.uk?(appointment_summary.country) ? nil : appointment_summary.country
end
```